### ISSUE TITLE
Minor corrections

### ISSUE THREAD

**Issue by hbeni** [HHS81/c182s#296]
Today i had the opportunity to sit in a c182t and observed some things applying to the 182s too:

- [x] The davtron was "off" when battery switch was off, however the ET timer was persistent. Until now i thought from the manual that the little internal battery fused the whole device while in reality it seems that it only drives the quarz clock (like a CMOS battery in a computer). (done in branch for #296 because of dependencies)
- [x] oil pressure in idle is already inside the green arc. Oil pressure is not directly RPM tied and needs some brief initial time after start to show values (as mentioned in the manual, this is depending on the oil temp). We should simulate both effects.
- [x] Starting then engine (in warm weather with warm engine!) resulted in a small puff of dark smoke from both cowlings.
- [x] correct tooltip update at pitot and cabin heat/air levers

**Comment by hbeni** [comment]
The oil-takes-some-time-to-show-values could be simulated quite easily with delaying the already existing oil pump -norm property; for example multiply it with a factor-mapping table depending on oil temp. This should simulate cold oil quite reasonably. The manual says that the pump needs some time to get pressure because of viscosity of cold oil.

**Comment by hbeni** [comment]
Smoke is implemented; however currently still lacking texture. I don't know why, but asked on the forums: https://forum.flightgear.org/viewtopic.php?f=47&t=34771

**Comment by wlbragg** [comment]
@hbeni I commented out all but the exhaust-smoke-left effect and called it exclusively in the Models/c182s.xml file

~~~
<model>
  <path>Aircraft/c182s/Models/Effects/c182_enginesmoke.xml</path>
  <name>exhaust-smoke-left</name>
  <offsets>
    <x-m>-0.60</x-m>
    <y-m> 0.00</y-m>
    <z-m>-0.10</z-m>
  </offsets>
</model>
~~~

You get your individual round puffs then. I think it is a combination of calling it all at once.

**Comment by hbeni** [comment]
@wlbragg Fixed and implemented, it seems only one effect is allowed per definition file.
![image](https://user-images.githubusercontent.com/13608602/45304717-f6d81900-b518-11e8-9caf-194b94089ce7.png)

**Comment by wlbragg** [comment]
Sorry for just now seeing this. There is no problem calling multiple particle emitters in the same file if you have the shooters set up to account for the initial starting position of the call from Models/c182s.xml. Notice now how you are setting each of their initial positions. Let say that position was dead center between the two exhausts, each emitter would have to be offset then to get the correct position for each one. There is nothing wrong either with breaking it up like your doing, it makes it easier to read and adjust.

**Comment by hbeni** [comment]
Should be ready now.
I decided against implementing the "oil not showing pressure some time after start" because i have no real world numbers on how long it takes.

**Comment by hbeni** [comment]
should be closed by #297 some time ago
